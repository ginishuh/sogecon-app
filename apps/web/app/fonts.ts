import localFont from 'next/font/local';

// 전역 본문/헤딩 공통 폰트 — self-hosted (next/font)
// - Inter(variable) + 시스템 폰트 폴백
// - next/font는 빌드 시 폰트를 프로젝트에 포함하므로 테스트 환경에서도 안전합니다.
export const fontSans = localFont({
  src: [
    {
      path: '../public/fonts/Inter-Variable.woff2',
      style: 'normal',
      // Variable font axis range for CSS. Note: narrowing this range
      // does NOT reduce the font file size for a single variable woff2.
      // To truly reduce bytes, switch to static weights (e.g., 400/500/700)
      // with multiple woff2 files or introduce subsetting.
      weight: '100 900',
    },
  ],
  variable: '--font-sans',
  display: 'swap',
  preload: true,
});
