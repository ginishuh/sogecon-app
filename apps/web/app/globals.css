@tailwind base;
@tailwind components;
@tailwind utilities;

:root {
  color-scheme: light;
  --surface-background: #ffffff; /* 화면 전체 배경: 흰색으로 통일 */
  --surface-card: #ffffff;
  --surface-muted: #fdf8f2;
  --text-primary: #1f242d;
  --text-muted: #626b79;
  --border-soft: #e3d7cf;
  --focus-ring: #f4a259;
  --brand-primary: #8a1e2d;
  --brand-primary-dark: #6c1722;
  --brand-accent: #f4a259;
}

body {
  min-height: 100vh;
  margin: 0;
  background: var(--surface-background);
  color: var(--text-primary);
  /* 전역 폰트: next/font 변수를 우선 사용(시스템 폰트로 안전한 폴백) */
  font-family: var(--font-sans), -apple-system, BlinkMacSystemFont, 'Segoe UI', system-ui, sans-serif;
  /* 타이포 스케일 — 모바일 기준 body 15px → md 이상 16px */
  font-size: 0.9375rem; /* 15px */
  line-height: 1.6;
  display: flex;
  flex-direction: column;
}

main {
  margin: 0 auto;
  width: 100%;
  max-width: 72rem;
  padding: 3rem 1.5rem 4rem;
  display: flex;
  flex-direction: column;
  gap: 4rem;
}

main:focus {
  outline: none;
}

@media (min-width: 768px) {
  body { font-size: 1rem; } /* 16px */
  main { padding: 4rem 2.5rem 5rem; }
}

/* 링크 상태 일관 (AA 대비 유지) */
a { color: theme(colors.brand.700); transition: color 150ms ease; text-decoration: none; }
a:visited { color: theme(colors.brand.600); }
a:hover { color: theme(colors.brand.800); text-decoration: underline; }
a:active { color: theme(colors.brand.900); }
a:focus-visible { outline: none; text-decoration: underline; }

/* 포커스 링 — 토큰 기반 ring */
:where(button, [role="button"], a, input, select, textarea):focus-visible {
  outline: none;
  @apply ring-2 ring-brand-400 ring-offset-2 ring-offset-surface;
}

.skip-link {
  position: absolute;
  top: 1rem;
  left: 1.5rem;
  padding: 0.75rem 1.5rem;
  border-radius: 9999px;
  background: var(--brand-primary);
  color: #ffffff;
  transform: translateY(-200%);
  transition: transform 150ms ease;
  z-index: 50;
}

.skip-link:focus {
  transform: translateY(0);
  outline: none;
  box-shadow: 0 0 0 4px rgba(244, 162, 89, 0.55);
}

.site-footer {
  margin: 0 auto;
  width: 100%;
  max-width: 72rem;
  padding: 2rem 1.5rem 3rem;
  color: var(--text-muted);
  font-size: 0.75rem;
}

@media (min-width: 768px) {
  .site-footer {
    padding: 2rem 2.5rem 4rem;
  }
}

@layer components {
  /* 시맨틱 헤딩(모바일 기준) — md 이상에서 점진 확대 */
  h1 { @apply text-h1 font-heading text-text-primary; }
  h2 { @apply text-h2 font-heading text-text-primary; }
  h3 { @apply text-h3 font-heading text-text-primary; }
  @media (min-width: 768px) {
    h1 { @apply text-4xl; } /* 약 32px */
    h2 { @apply text-2xl; } /* 약 24px */
    h3 { @apply text-xl; }  /* 약 20px */
  }
  .about-page {
    @apply flex flex-col gap-16;
  }

  .about-hero {
    @apply grid gap-8 rounded-3xl bg-white px-6 py-10 shadow-soft md:grid-cols-[1.3fr_1fr] md:px-10;
  }

  .about-hero__text {
    @apply flex flex-col gap-4;
  }

  .about-hero__eyebrow {
    @apply text-xs font-semibold uppercase tracking-widest text-neutral-muted;
  }

  .about-hero__title {
    @apply font-heading text-display-xl text-brand-primary;
  }

  .about-hero__description {
    @apply text-lg text-neutral-muted;
  }

  .about-hero__image {
    @apply flex items-center justify-center;
  }

  .about-section {
    @apply grid gap-8 rounded-3xl bg-white px-6 py-10 shadow-sm md:grid-cols-[1.1fr_0.9fr] md:px-10;
  }

  .about-section--stacked {
    @apply flex flex-col gap-10 bg-brand-surface shadow-none;
  }

  .about-section--timeline {
    @apply bg-white;
  }

  .about-section--highlight {
    @apply bg-white shadow-soft;
  }

  .about-section__content {
    @apply flex flex-col gap-4 text-base leading-7 text-neutral-muted;
  }

  .about-section__heading {
    @apply font-heading text-2xl font-semibold text-brand-primary md:text-3xl;
  }

  .about-section__figure {
    @apply flex items-center justify-center;
  }

  .about-section__list {
    @apply grid gap-6 md:grid-cols-3;
  }

  .about-section__card {
    @apply flex h-full flex-col gap-3 rounded-2xl border border-neutral-border bg-white p-6 shadow-sm;
  }

  .about-section__card-title {
    @apply font-heading text-lg font-semibold text-brand-primary;
  }

  .about-section__dl {
    @apply mt-2 flex flex-col gap-2 text-sm;
  }

  .about-section__dl dt {
    @apply text-xs font-medium uppercase tracking-wide text-neutral-muted;
  }

  .about-section__dl dd {
    @apply text-neutral-ink;
  }

  .about-section__card p {
    @apply text-sm text-neutral-muted;
  }

  .about-timeline {
    @apply relative flex flex-col gap-10;
  }

  .about-timeline::before {
    @apply absolute left-3 top-0 h-full w-0.5 bg-neutral-border content-[''];
  }

  .about-timeline__item {
    @apply relative flex gap-5 pl-8;
  }

  .about-timeline__marker {
    @apply absolute left-[-3px] top-1.5 h-4 w-4 rounded-full border-4 border-brand-primary bg-white;
  }

  .about-timeline__content {
    @apply flex flex-col gap-1 rounded-xl bg-white p-5 shadow-sm;
  }

  .about-timeline__year {
    @apply text-xs font-semibold uppercase tracking-wider text-brand-primary;
  }

  .about-timeline__title {
    @apply font-heading text-lg font-semibold text-neutral-ink;
  }

  .about-contact-list {
    @apply grid gap-4 md:grid-cols-3;
  }

  .about-contact-list__item {
    @apply rounded-xl border border-neutral-border bg-white p-5 shadow-sm;
  }

  .about-contact-list__term {
    @apply text-xs font-semibold uppercase tracking-widest text-neutral-muted;
  }

  .about-contact-list__description {
    @apply mt-1 text-sm text-neutral-ink;
  }

  .home-hero {
    /* 모바일 높이 축소: 여백·격자 간격 축소, md 이상은 기존 값 유지 */
    @apply grid gap-6 rounded-3xl bg-white/95 px-5 py-8 shadow-soft md:grid-cols-[1.2fr_1fr] md:px-12 md:py-12 md:gap-10;
  }

  .home-hero__eyebrow {
    @apply text-xs font-semibold uppercase tracking-widest text-neutral-muted;
  }

  .home-hero__title {
    /* 모바일 타이틀 크기 하향: 텍스트 줄바꿈 최소화 */
    @apply font-heading text-h1 text-brand-primary md:text-display-xl;
  }

  .home-hero__description {
    /* 본문 크기 축소(15px 스케일) */
    @apply text-body leading-7 text-neutral-muted;
  }

  .home-hero__actions {
    @apply mt-6 flex flex-wrap gap-3;
  }

  .home-hero__cta {
    @apply inline-flex items-center justify-center rounded-full bg-brand-primary px-6 py-3 text-sm font-semibold text-white transition hover:bg-brand-primaryDark focus:outline-none focus-visible:ring-4 focus-visible:ring-brand-accent;
  }

  .home-hero__secondary {
    @apply inline-flex items-center justify-center rounded-full border border-neutral-border px-6 py-3 text-sm font-semibold text-neutral-ink transition hover:border-brand-primary hover:text-brand-primary focus:outline-none focus-visible:ring-4 focus-visible:ring-brand-accent;
  }

  .home-hero__image {
    @apply hidden self-center rounded-3xl bg-brand-surface p-4 shadow-sm md:block;
  }

  .home-card-grid {
    @apply grid gap-6 md:grid-cols-2 lg:grid-cols-3;
  }

  .home-card {
    @apply flex h-full flex-col justify-between rounded-2xl border border-neutral-border bg-white p-6 shadow-sm transition hover:-translate-y-1 hover:shadow-soft focus-within:ring-4 focus-within:ring-brand-accent;
  }

  .home-card__title {
    @apply font-heading text-xl font-semibold text-neutral-ink;
  }

  .home-card__description {
    @apply mt-3 text-sm text-neutral-muted;
  }

  .home-card__meta {
    @apply mt-6 flex items-center justify-between text-xs uppercase tracking-widest text-neutral-muted;
  }
}

/* 홈: Quick Actions (2×2) */
@layer components {
  .home-quick-actions { @apply px-0 py-0 bg-transparent shadow-none rounded-none; }

  .home-quick-actions__grid { @apply grid grid-cols-3 gap-3 md:gap-4 lg:grid-cols-6; }

  .home-quick-actions__item {
    @apply flex flex-col items-center justify-center gap-2 rounded-2xl px-3 py-4 text-center shadow-sm transition hover:-translate-y-0.5 hover:shadow-soft focus-visible:ring-4 focus-visible:ring-brand-accent md:px-4;
    aspect-ratio: 1 / 1; /* 정사각형 카드 */
  }

  .home-quick-actions__icon { @apply mb-2 inline-flex h-10 w-10 items-center justify-center text-white; }

  .home-quick-actions__label { @apply font-menu text-sm font-semibold tracking-tight leading-none md:text-base truncate whitespace-nowrap max-w-full; }

  .home-quick-actions__desc { @apply hidden; }
}

/* 빠른 실행: 색상 변형(전체 카드 배경 컬러 + 텍스트 흰색) */
@layer components {
  .home-quick-actions__item[class*="qa-"] { @apply border-0 text-white; }
  .home-quick-actions__item.qa-rose { @apply bg-rose-700 hover:bg-rose-600; }
  .home-quick-actions__item.qa-emerald { @apply bg-emerald-700 hover:bg-emerald-600; }
  .home-quick-actions__item.qa-sky { @apply bg-sky-700 hover:bg-sky-600; }
  .home-quick-actions__item.qa-amber { @apply bg-amber-700 hover:bg-amber-600; }
  .home-quick-actions__item.qa-indigo { @apply bg-indigo-700 hover:bg-indigo-600; }
  .home-quick-actions__item.qa-slate { @apply bg-slate-800 hover:bg-slate-700; }
  .home-quick-actions__item[class*="qa-"] .home-quick-actions__label { @apply text-white; }
  .home-quick-actions__item[class*="qa-"] .home-quick-actions__desc { @apply text-white/90; }
}

/* 홈: 히어로 이미지 — 모바일 숨김 유지, md 이상에서만 표시 */
@layer components {
  .home-hero__image { @apply relative hidden self-center overflow-hidden rounded-3xl bg-brand-surface p-0 shadow-sm md:block md:p-0; }
  .home-hero__image::after { content: ''; position: absolute; inset: 0; @apply rounded-3xl; background: linear-gradient(to top, rgba(0,0,0,0.25), rgba(0,0,0,0.08)); pointer-events: none; }
}

/* 홈: 미니멀 히어로(이미지 + 좌하단 캡션 1줄) */
@layer components {
  .home-hero--image-only { @apply overflow-hidden rounded-3xl bg-transparent p-0 shadow-none; }
  .home-hero__figure { @apply relative block overflow-hidden rounded-3xl; aspect-ratio: 16/9; }
  .home-hero__media { @apply absolute inset-0 h-full w-full object-cover; }
  /* 오버레이 제거, 텍스트 영역만 바 처리 */
  .home-hero__figure::after { content: none; }
  /* 하단 캡션 — 불투명도 높은 검은색 바 + 흰색 볼드 텍스트 */
  .home-hero__caption { @apply absolute bottom-3 left-3 max-w-[88%] truncate rounded-lg bg-black/85 px-3 py-1.5 text-sm font-bold text-white md:text-base; }
}

/* 홈: 히어로 캐러셀 */
@layer components {
  .hero-carousel { @apply relative; }
  .hero-track { @apply flex transition-transform duration-500; }
  .hero-slide { @apply min-w-full; }
  .hero-arrow { @apply absolute top-1/2 z-10 -translate-y-1/2 rounded-full bg-black/40 p-2 text-white backdrop-blur hover:bg-black/60 focus:outline-none focus-visible:ring-4 focus-visible:ring-brand-accent; }
  .hero-arrow--prev { @apply left-2 md:left-3; }
  .hero-arrow--next { @apply right-2 md:right-3; }
  .hero-dots { @apply mt-2 flex items-center justify-center gap-2; }
  .hero-dot { @apply h-2.5 w-2.5 rounded-full bg-neutral-muted/60; }
  .hero-dot.is-active { @apply bg-brand-primary; }
}
