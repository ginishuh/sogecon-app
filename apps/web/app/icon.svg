<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 64">
  <defs>
    <linearGradient id="g" x1="0" y1="0" x2="1" y2="1">
      <stop offset="0%" stop-color="#0ea5e9"/>
      <stop offset="100%" stop-color="#2563eb"/>
    </linearGradient>
  </defs>
  <rect x="2" y="2" width="60" height="60" rx="12" fill="url(#g)"/>
  <path d="M21 46l11-28h0l11 28h-6l-2.2-6H29.2L27 46h-6zm11-20l-3.4 9h6.8L32 26z" fill="#ffffff"/>
</svg>

