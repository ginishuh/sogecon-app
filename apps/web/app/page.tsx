import Link from 'next/link';

const apiBase =
  process.env.NEXT_PUBLIC_WEB_API_BASE ?? process.env.WEB_API_BASE ?? 'http://localhost:3001';

async function fetchHealth() {
  try {
    const res = await fetch(`${apiBase}/healthz`, {
      cache: 'no-store',
      next: { revalidate: 0 }
    });

    if (!res.ok) {
      return { ok: false, status: res.status };
    }

    const data = (await res.json()) as { ok: boolean };
    return { ok: Boolean(data.ok), status: res.status };
  } catch (error) {
    return { ok: false, status: 503 };
  }
}

export default async function HomePage() {
  const health = await fetchHealth();

  return (
    <section className="space-y-6">
      <p>
        공개 동문 애플리케이션의 홈 화면입니다. 게시글과 행사는 로컬 API에서 제공되며, 추후 인증과
        권한 기능이 추가될 예정입니다.
      </p>
      <div className="rounded-md border border-slate-200 bg-white p-4">
        <h2 className="text-lg font-semibold">API 상태</h2>
        <p className="mt-2 text-sm text-slate-600">
          {health.ok ? '연결됨 ✅' : `연결 안 됨 ⚠️ (status ${health.status})`}
        </p>
        <p className="mt-1 text-xs text-slate-500">기본 엔드포인트: {apiBase}</p>
      </div>
      <div className="grid gap-4 md:grid-cols-2">
        <Link className="rounded border border-slate-200 bg-white p-4 shadow-sm" href="/posts">
          <h3 className="font-semibold">게시글</h3>
          <p className="mt-1 text-sm text-slate-600">공지와 소식을 모아볼 공간입니다.</p>
        </Link>
        <Link className="rounded border border-slate-200 bg-white p-4 shadow-sm" href="/events">
          <h3 className="font-semibold">행사</h3>
          <p className="mt-1 text-sm text-slate-600">정기 모임 및 RSVP 관리를 위한 자리입니다.</p>
        </Link>
      </div>
    </section>
  );
}
