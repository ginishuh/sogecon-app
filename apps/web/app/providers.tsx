"use client";

import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
import { ReactNode, useState } from 'react';
import { ToastProvider } from '../components/toast';

export function Providers({ children }: { children: ReactNode }) {
  // QueryClient는 클라이언트에서 1회 생성
  const [client] = useState(() => new QueryClient());
  return (
    <QueryClientProvider client={client}>
      <ToastProvider>{children}</ToastProvider>
    </QueryClientProvider>
  );
}
