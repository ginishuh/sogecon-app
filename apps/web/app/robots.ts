import type { MetadataRoute } from 'next';

import { siteConfig } from '../lib/site';

export default function robots(): MetadataRoute.Robots {
  const baseUrl = new URL(siteConfig.url);
  return {
    rules: [
      {
        userAgent: '*',
        allow: '/',
        disallow: ['/admin', '/posts/new', '/events/new']
      }
    ],
    sitemap: [`${baseUrl.origin}/sitemap.xml`],
    host: baseUrl.host
  };
}
