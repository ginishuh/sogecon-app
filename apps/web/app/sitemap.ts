import type { MetadataRoute } from 'next';

import { siteConfig } from '../lib/site';

const staticPaths = [
  '/',
  '/about/greeting',
  '/about/org',
  '/about/history',
  '/directory',
  '/faq',
  '/privacy',
  '/terms',
  '/posts',
  '/events',
  '/offline'
];

export default function sitemap(): MetadataRoute.Sitemap {
  const base = siteConfig.url.replace(/\/$/, '');
  const now = new Date();
  return staticPaths.map((path) => ({
    url: `${base}${path === '/' ? '' : path}`,
    lastModified: now,
    changeFrequency:
      path === '/'
        ? 'weekly'
        : ['/posts', '/events', '/directory'].includes(path)
          ? 'weekly'
          : ['/faq', '/privacy', '/terms'].includes(path)
            ? 'yearly'
            : 'monthly',
    priority: path === '/' ? 1 : 0.6
  }));
}
