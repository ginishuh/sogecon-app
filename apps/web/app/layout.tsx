import './globals.css';
import type { Metadata } from 'next';
import type { ReactNode } from 'react';

import { ServiceWorkerRegister } from './sw-register';
import { Providers } from './providers';
import { SiteHeader } from '../components/site-header';
import { Analytics } from '../components/analytics';
import { siteConfig, ogImage } from '../lib/site';
import { fontSans } from './fonts';

const metadataUrl = new URL(siteConfig.url);
const ogImageUrl = new URL(ogImage.path, siteConfig.url);

export const metadata: Metadata = {
  metadataBase: metadataUrl,
  applicationName: siteConfig.shortName,
  category: 'organization',
  title: {
    default: `${siteConfig.name} 웹 서비스`,
    template: `%s | ${siteConfig.name}`
  },
  description: siteConfig.description,
  keywords: ['총원우회', '서강대학교 경제대학원', '동문 디렉터리', '총회', '행사', 'FAQ'],
  authors: [{ name: siteConfig.name }],
  creator: siteConfig.name,
  publisher: siteConfig.name,
  openGraph: {
    type: 'website',
    locale: 'ko_KR',
    siteName: siteConfig.name,
    title: `${siteConfig.name} 웹 서비스`,
    description: siteConfig.description,
    url: siteConfig.url,
    images: [
      {
        url: ogImageUrl.toString(),
        width: 1200,
        height: 630,
        alt: ogImage.alt,
        type: 'image/png'
      }
    ]
  },
  twitter: {
    card: 'summary_large_image',
    title: `${siteConfig.name} 웹 서비스`,
    description: siteConfig.description,
    images: [ogImageUrl.toString()]
  },
  alternates: {
    canonical: siteConfig.url,
    languages: {
      ko: siteConfig.url
    }
  },
  robots: {
    index: true,
    follow: true,
    googleBot: {
      index: true,
      follow: true
    }
  }
};

export default function RootLayout({
  children
}: {
  children: ReactNode;
}) {
  return (
    <html lang="ko" className={`${fontSans.variable} antialiased`}>
      <body>
        <a className="skip-link" href="#main-content">
          본문 바로가기
        </a>
        {/* Dev에서 SW가 RSC 스트림을 끊는 것을 피하기 위해 prod/NEXT_PUBLIC_ENABLE_SW=1 에서만 렌더 */}
        {(process.env.NODE_ENV === 'production' || process.env.NEXT_PUBLIC_ENABLE_SW === '1') && (
          <ServiceWorkerRegister />
        )}
        <Providers>
          <Analytics />
          <SiteHeader />
          <main id="main-content" role="main" tabIndex={-1}>
            {children}
          </main>
          <footer className="site-footer" role="contentinfo">
            Public alumni app scaffold — local use only for now.
          </footer>
        </Providers>
      </body>
    </html>
  );
}
