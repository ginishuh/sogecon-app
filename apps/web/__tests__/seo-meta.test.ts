import { describe, expect, it } from 'vitest';

import { metadata } from '../app/layout';
import sitemap from '../app/sitemap';
import robots from '../app/robots';

describe('SEO metadata configuration', () => {
  it('declares title template and open graph baseline', () => {
    expect(metadata.title).toMatchObject({
      default: '서강대 경제대학원 총원우회',
      template: expect.stringContaining('%s')
    });
    expect(metadata.openGraph?.images?.[0]?.url).toContain('default-og.svg');
    expect(metadata).toMatchSnapshot();
  });

  it('includes about routes in sitemap', () => {
    const entries = sitemap();
    const urls = entries.map((entry) => entry.url);
    expect(urls).toEqual(expect.arrayContaining([
      expect.stringContaining('/about/greeting'),
      expect.stringContaining('/about/org'),
      expect.stringContaining('/about/history')
    ]));
  });

  it('allows crawling and references sitemap in robots config', () => {
    const config = robots();
    expect(config.rules?.[0]?.allow).toBe('/');
    expect(config.sitemap?.[0]).toContain('/sitemap.xml');
    expect(config.host).toBeDefined();
  });
});
