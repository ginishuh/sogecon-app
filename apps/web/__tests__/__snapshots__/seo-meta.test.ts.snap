// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SEO metadata configuration > declares title template and open graph baseline 1`] = `
{
  "alternates": {
    "canonical": "https://alumni.sogang-econ.kr",
    "languages": {
      "ko": "https://alumni.sogang-econ.kr",
    },
  },
  "applicationName": "총동문회",
  "authors": [
    {
      "name": "서강대 경제대학원 총동문회",
    },
  ],
  "category": "organization",
  "creator": "서강대 경제대학원 총동문회",
  "description": "서강대학교 경제대학원 총동문회 공식 웹 — 공지, 행사, 디렉터리, FAQ를 한 화면에서 확인하고 동문 네트워크와 바로 연결하세요.",
  "keywords": [
    "총동문회",
    "서강대학교 경제대학원",
    "동문 디렉터리",
    "총회",
    "행사",
    "FAQ",
  ],
  "metadataBase": "https://alumni.sogang-econ.kr/",
  "openGraph": {
    "description": "서강대학교 경제대학원 총동문회 공식 웹 — 공지, 행사, 디렉터리, FAQ를 한 화면에서 확인하고 동문 네트워크와 바로 연결하세요.",
    "images": [
      {
        "alt": "서강대 경제대학원 총동문회 웹 런치 대표 이미지",
        "height": 630,
        "type": "image/png",
        "url": "https://alumni.sogang-econ.kr/og-default.png",
        "width": 1200,
      },
    ],
    "locale": "ko_KR",
    "siteName": "서강대 경제대학원 총동문회",
    "title": "서강대 경제대학원 총동문회 웹 서비스",
    "type": "website",
    "url": "https://alumni.sogang-econ.kr",
  },
  "publisher": "서강대 경제대학원 총동문회",
  "robots": {
    "follow": true,
    "googleBot": {
      "follow": true,
      "index": true,
    },
    "index": true,
  },
  "title": {
    "default": "서강대 경제대학원 총동문회 웹 서비스",
    "template": "%s | 서강대 경제대학원 총동문회",
  },
  "twitter": {
    "card": "summary_large_image",
    "description": "서강대학교 경제대학원 총동문회 공식 웹 — 공지, 행사, 디렉터리, FAQ를 한 화면에서 확인하고 동문 네트워크와 바로 연결하세요.",
    "images": [
      "https://alumni.sogang-econ.kr/og-default.png",
    ],
    "title": "서강대 경제대학원 총동문회 웹 서비스",
  },
}
`;
